static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -425,6 +426,17 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
     }
   }
 
+  /**
+   * Register listeners so that any active pins are deselected when a user clicks
+   * on the search bar
+   */
+  setupSearchBarListeners () {
+    const searchBarForm = this._container.querySelector('.yxt-SearchBar-form');
+    searchBarForm && searchBarForm.addEventListener('click', () => {
```

Comment:
I notice we no longer have a `focus` listener in this PR. Does that have any WCAG implications?

Reply:
I removed it because
- It's not necessary for us to have it to be WCAG compliant, and its a rare use case where someone is on mobile and using keyboard
- I felt that it would make the code a little more confusing since I would have to add focus listeners to two additional elements

However I can support it if you'd like

Reply:
Updated to include focus listeners!